package.json

```diff
@@ -32,6 +34,8 @@
     "@types/node": "17.0.21",
     "@types/shelljs": "0.8.11",
     "@types/uuid": "8.3.4",
+    "@types/yargs": "^17.0.10",
+    "prettier": "^2.6.0",
```

Comment:
i think we can remove `prettier` for now add it later when we actually integrate it into CI etc

Reply:
I've added it because my workflow involves trying to add as few global packages to my namespace as possible (to avoid conflicting versions in my global namespace). Adding `prettier` here means I can invoke prettier from dev dependencies vs having to install it globally.